pkgs/tools/virtualization/nixos-nspawn/nixos-nspawn.py

```diff
@@ -0,0 +1,512 @@
+#!/usr/bin/env nix-shell
```

Comment:
running `nixos-nspawn` shows no output may show the help then? (and also return an error code)?

Reply:
Will take a look. The current implementation isn't very nice and parts of the script should be refactored anyways.